examples/src/main/java/com/opengamma/strata/examples/finance/CalibrationCheckExample.java

```diff
@@ -203,7 +208,7 @@ public static void performance_calibration_pricing() {
       for (CurveNode node : nodes) {
         if (!(node instanceof IborFixingDepositCurveNode)) {
           // IborFixingDeposit is not a real trade, so there is no appropriate comparison
-          trades.add(node.trade(VALUATION_DATE, ObservableValues.ofIdMap(quotes)));
+          trades.add(node.trade(VALUATION_DATE, MarketData.builder().addValues(quotesByKey).build()));
```

Comment:
I did consider it but there are only 2 instances of this pattern in the whole codebase so I wasn't sure it was worth adding something to the API.